code/datums/controllers/throwing_controls.dm

```diff
@@ -121,10 +123,18 @@ var/global/datum/controller/throwing/throwing_controller = new
 					new /obj/effects/explosion/dangerous(next)
 					end_throwing = FALSE
 					thr.throw_type = THROW_NORMAL
+
+			if (thr.throw_type == THROW_PHASE && thr.get_throw_travelled() > 2)
+				thr.throw_type = THROW_NORMAL
+				thing.event_handler_flags &= ~MOVE_NOCLIP
```

Comment:
My only concern is, what if it started with no clip, but that isn't going to happen in this usage so... Doesn't really matter IMO

Reply:
Do you think it's worth it to do add initial flag checking?

Reply:
I guess just doing initial() would be fine yeah